### PR TITLE
Weapon list tablets

### PR DESCRIPTION
**Pull request by peter-budo**
@karllindmark 

 • Removed unused class WeaponStats
 • Added layout specific to 10" tablet
 • Fixed vehicle stats list back scrolling issue
 • Update changelog

device-2014-05-26-175750
device-2014-05-26-180434
)

### CODE REVIEW

File: BF4Intel/src/main/java/com/ninetwozero/bf4intel/utils/NumberFormatter.java [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
 public class NumberFormatter {
     public static String format(final long number) {
         return String.format(Locale.getDefault(), "%,d", number);
     }
+
     public static String format(final double number) {
         return String.format(Locale.getDefault(), "%.2f", number);
     }
+
+    public static String format(final BigDecimal number) {
```
Comment (karllindmark) [inline BF4Intel/src/main/java/com/ninetwozero/bf4intel/utils/NumberFormatter.java]: Any specific reason for the introduction of the `BigDecimal` class? :-)

**Comment by karllindmark** [comment]
@peter-budo: I'm a bit conflicted when it comes to the layout. I like the thought, but there's something bugging me and I can't put my finger on it. Let me get back to you on that end!

File: BF4Intel/src/main/java/com/ninetwozero/bf4intel/utils/NumberFormatter.java (hunk 1 above)
Comment (peter-budo) [inline BF4Intel/src/main/java/com/ninetwozero/bf4intel/utils/NumberFormatter.java]: Forgotten artifact. I was trying to do precision calculation, but I run in arithmetic exception. Hence the stupid way of multiply kills by 100, store value, and then before displaying it divide by 100 to get correct percentage.
I will delete it, once by Mac

**Comment by peter-budo** [comment]
From my point of view, I guess it is time to split weapon and vehicles layout as this is not helping on 10" tablet. There is a gap because accuracy is not available to vehicles

**Comment by karllindmark** [comment]
@peter-budo: +1 on making the XML files separate (fragment_weapon_list and fragment_vehicle_list I suppose). :-)

**Comment by karllindmark** [comment]
@peter-budo: I haven't had a chance to try it on my 10" yet, but looks like expected on the phone (ie, no side-effects).

**Comment by peter-budo** [comment]
@karllindmark small push. Seems that yesterday I did not finished renaming "item_image" to weapon and vehicle image. Also bellow is latest vehicles screenshot. If you happy then merge :wink:  
device-2014-05-28-090204

**Pull request merged**